website/docs/basics/tips-tricks.md

```diff
@@ -0,0 +1,69 @@
+---
+id: tips-tricks
+title: Tips & Tricks
+---
+
+As we are making our projects using this setup we are going to share our tips and tricks for faster development, easier debugging and some cool ideas we had along the way.
```

Comment:
```suggestion
Since we are creating our projects using this setup, we are going to share our tips and tricks for faster development, easier debugging and some cool ideas we had along the way.
```